### PR TITLE
Issue #244: Adding config parameter to turn on/off 'Printing debug info about browser'

### PR DESCRIPTION
**Pull request by SudhakaraPeram**
Issue #244: Adding config parameter to turn on/off 'Printing debug info
about browser'

### CODE REVIEW

**Comment by SudhakaraPeram** [comment]
@mach6, Please review my pull request when you get time. Thanks!

File: client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/WebDriverFactory.java [hunk 1]
```diff
@@ -84,7 +86,9 @@ public static RemoteWebDriver createInstance(BrowserFlavors browser) {
         logger.log(Level.FINE, "Spawning a browser with the following capabilities: "
                 + DriverFactoryHelper.showCapabilities(capability));
         driver = createDriverInstance(DriverFactoryHelper.getURL(), capability);
-        DriverFactoryHelper.printDebugInfoForUser(driver);
+        if (Boolean.parseBoolean(Config.getConfigProperty(ConfigProperty.PRINT_DEBUG_INFO))) {
```
Comment (mach6) [inline client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/WebDriverFactory.java]: There is a method getBoolConfigProperty in Config which would be better to use here.

**Comment by mach6** [comment]
@SudhakaraPeram thank you. This looks good. I'm traveling for the next few days. 
@sebady or @sundaramrajendran can finish this review and merge it.

**Comment by SudhakaraPeram** [comment]
Updated to Config.getBoolConfigProperty as per @mach6 suggestion.
@sebady, @sundaramrajendran: Can anyone of you review the code and merge. Thanks!

**Pull request merged**